docker-plugin/src/main/java/com/nirima/jenkins/plugins/docker/DockerTemplateBase.java

```diff
@@ -98,6 +97,43 @@ protected Object readResolve() {
         return this;
     }
 
+    protected Bind[] volumesFromString(String volumesString){
+        List<Bind> bindList = new ArrayList<Bind>();
+        for( String volume : splitAndFilterEmpty(volumesString)){
+            List<String> volumeSpecs = Splitter.on(":").omitEmptyStrings().splitToList(volume);
```

Comment:
I feel that you should better do it with regexp and optional group matching. 
And it will be very good to move such functionality to docker-java....